### ISSUE TITLE
Failed to exec iptables-restore for iptables >= 1.6.2

### ISSUE THREAD

**Issue by gagara** [kubernetes/minikube#2647]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`):
minikube version: v0.25.2
- **OS** (e.g. from /etc/os-release):
Gentoo Linux
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`):
    "DriverName": "none",
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`):
??
- **Install tools**:
- **Others**:

**What happened**:
When creating service with `kubectl` port on host machine is not opened. In minikube logs I have:
```
E0324 16:54:58.207422    5943 proxier.go:1667] Failed to execute iptables-restore: exit status 1 (iptables-restore: invalid option -- '5'
Try `iptables-restore -h' for more information.
```
**What you expected to happen**:
When service created port on host machine should be opened and exposed to hello-minikube pod. 

**How to reproduce it** (as minimally and precisely as possible):
Follow hello-minikube guide

**Output of `minikube logs` (if applicable)**:
```
E0324 16:54:58.207422    5943 proxier.go:1667] Failed to execute iptables-restore: exit status 1 (iptables-restore: invalid option -- '5'
Try `iptables-restore -h' for more information.
```
**Anything else do we need to know**:
```
$ iptables-restore --version
iptables-restore v1.6.2
```
```
$ iptables-restore -h
Usage: iptables-restore [-c] [-v] [-V] [-t] [-h] [-n] [-w secs] [-W usecs] [-T table] [-M command]
	   [ --counters ]
	   [ --verbose ]
	   [ --version]
	   [ --test ]
	   [ --help ]
	   [ --noflush ]
	   [ --wait=<seconds>
	   [ --wait-interval=<usecs>
	   [ --table=<TABLE> ]
	   [ --modprobe=<command> ]
```
```
# iptables-restore -w5
iptables-restore: invalid option -- '5'
Try `iptables-restore -h' for more information.
```
Maybe the same as https://github.com/kubernetes/kubernetes/issues/56056?

**Comment by attila123** [comment]
Note: e.g. AntergOS Linux also has iptables v1.6.2 now. If someone wants to try this, AntergOS is much easier to install (for most people). Or probably manually updating iptables is also an option (did not check its dependencies).

**Comment by dlorenc** [comment]
This looks like it's working as intended - the none driver for minikube requires that you manually install the correct versions of all the dependencies. It looks like iptables >1.6.0 is known to not work with Kubernetes: https://github.com/kubernetes/kubernetes/issues/58956

You'll have to downgrade until a fix is available in k8s.

**Comment by dlorenc** [comment]
It looks like this might already be fixed in k8s 1.10.0. Which version did you try this in?

**Comment by gagara** [comment]
Yes. Seems like it is fixed in latest version of k8s. So, I would close this one. Thx.
